src/NzbDrone.Core.Test/IndexerTests/TorrentRssIndexerTests/TorrentRssIndexerFixture.cs

```diff
@@ -151,5 +151,106 @@ public void should_parse_recent_feed_from_Doki()
             torrentInfo.Peers.Should().NotHaveValue();
             torrentInfo.Seeders.Should().NotHaveValue();
         }
+
+        [Test]
+        public void should_parse_recent_feed_from_BeyondHD()
+        {
+            GivenRecentFeedResponse("TorrentRss/BeyondHD.xml");
+
+            var releases = Subject.FetchRecent();
+
+            releases.Should().HaveCount(50);
+            releases.First().Should().BeOfType<TorrentInfo>();
+
+            var torrentInfo = releases.First() as TorrentInfo;
+
+            torrentInfo.Title.Should().Be("Tosh 0 S07E16 720p WEB-DL AAC2 0 H 264-pcsyndicate ");
+            torrentInfo.DownloadProtocol.Should().Be(DownloadProtocol.Torrent);
+            torrentInfo.DownloadUrl.Should().Be("https://beyondhd.me/download.php?torrent=39507&torrent_pass=0123456789abcdef0123456789abcdef");
+            torrentInfo.InfoUrl.Should().BeNullOrEmpty();
+            torrentInfo.CommentUrl.Should().BeNullOrEmpty();
+            torrentInfo.Indexer.Should().Be(Subject.Definition.Name);
+            torrentInfo.PublishDate.Should().Be((DateTime.Today + DateTime.Parse("04:27 PM").TimeOfDay).ToUniversalTime());
+            torrentInfo.Size.Should().Be(672210616);
+            torrentInfo.InfoHash.Should().BeNull();
+            torrentInfo.MagnetUrl.Should().BeNull();
+            torrentInfo.Peers.Should().Be(3);
+            torrentInfo.Seeders.Should().Be(3);
+
+            releases.ElementAt(1).Should().BeOfType<TorrentInfo>();
+            var secondTorrentInfo = releases.ElementAt(1) as TorrentInfo;
+
+            secondTorrentInfo.Title.Should().Be("Mr Robot S01E10 eps1 9 1 zer0-daY avi 1080p WEB-DL DD5 1 H 264-NTb ");
+            secondTorrentInfo.DownloadProtocol.Should().Be(DownloadProtocol.Torrent);
+            secondTorrentInfo.DownloadUrl.Should().Be("https://beyondhd.me/download.php?torrent=39504&torrent_pass=0123456789abcdef0123456789abcdef");
+            secondTorrentInfo.InfoUrl.Should().BeNullOrEmpty();
+            secondTorrentInfo.CommentUrl.Should().BeNullOrEmpty();
+            secondTorrentInfo.Indexer.Should().Be(Subject.Definition.Name);
+            secondTorrentInfo.PublishDate.Should().Be((DateTime.Today.Date - TimeSpan.FromDays(1) + DateTime.Parse("10:52 AM").TimeOfDay).ToUniversalTime());
+            secondTorrentInfo.Size.Should().Be(2254857830);
+            secondTorrentInfo.InfoHash.Should().BeNull();
+            secondTorrentInfo.MagnetUrl.Should().BeNull();
+            secondTorrentInfo.Peers.Should().Be(31);
+            secondTorrentInfo.Seeders.Should().Be(31);
+        }
+
+        [Test]
+        public void should_parse_recent_feed_from_HD4Free()
+        {
+            GivenRecentFeedResponse("TorrentRss/HD4Free.xml");
+
+            var releases = Subject.FetchRecent();
+
+            releases.Should().HaveCount(50);
+            releases.First().Should().BeOfType<TorrentInfo>();

```

Comment:
You can check if all the releases are TorrentInfo with this (instead of for each element): `releases.ForEach(s => s.Should().BeOfType<TorrentInfo>());`